### PR TITLE
Add RSA keys to launcher database

### PR DESCRIPTION
**Pull request by directionless**
This adds an RSA keypair to launcher's database. It will be used for some incoming features.

### CODE REVIEW

File: pkg/osquery/extension.go [hunk 1]
```diff
@@ -164,6 +170,10 @@ func NewExtension(client service.KolideService, db *bbolt.DB, opts ExtensionOpts
 		return nil, errors.Wrap(err, "creating DB buckets")
 	}
 
+	if err := SetupLauncherKeys(db); err != nil {
+		return nil, errors.Wrap(err, "setting up initial launcher keys")
```
Comment (James-Pickett) [inline pkg/osquery/extension.go:174]: nit / opinion: I feel like we should go ahead and switch to using `fmt.Errorf()` for all new error wrapping. I don't think the inconsistency in the file is really a problem.

File: pkg/osquery/extension.go [hunk 2]
```diff
@@ -219,8 +229,100 @@ func (e *Extension) getHostIdentifier() (string, error) {
 	return IdentifierFromDB(e.db)
 }
 
+func SetupLauncherKeys(db *bbolt.DB) error {
+	err := db.Update(func(tx *bbolt.Tx) error {
+
+		b, err := tx.CreateBucketIfNotExists([]byte(configBucket))
```
Comment (James-Pickett) [inline pkg/osquery/extension.go:235]: can we name this variable `bucket`? By the time I made it to the end of the function I forgot what `b` was =)

**Review by James-Pickett — APPROVED** [review]

**Review by James-Pickett — APPROVED** [review]
Dropped a couple of nits, but looks good!

File: pkg/osquery/extension.go (hunk 2 above)
Comment (directionless) [reply pkg/osquery/extension.go]: Sure

File: pkg/osquery/extension.go (hunk 1 above)
Comment (directionless) [reply pkg/osquery/extension.go]: Reasonable. I think I got these. Sometime we'll just need to change a bunch en-mass.

**Review by James-Pickett — APPROVED** [review]

**Pull request merged**